src/OpenSage.Game/Graphics/Cameras/Camera.cs

```diff
@@ -131,8 +131,10 @@ private void UpdateProjection()
 
         private void CreateProjection(out Matrix4x4 projection)
         {
+            var fov = GetVerticalFieldOfView();
+            fov = float.IsNaN(fov) ? 1 : fov; // .NET 8 no longer allows CreatePerspectiveFieldOfView to be called with NaN FOV
```

Comment:
I'm guessing this function never should have allowed NaN FOV, but did at one point? This seems to only happen on startup when the window size hasn't been updated and is 0x0. This feels like a sketchy hack to get it working so I'm open to alternatives, or maybe we fix it properly down the line.

Reply:
I'll investigate, may take some time though.

Reply:
Alright, I believe I've figured it out. I've rolled back the changes in this file.